envs/example/defaults.yml

```diff
@@ -102,6 +102,10 @@ nova:
   reserved_host_disk_mb: 50
   enable_ssh: true
   preallocate_images: none
+  haproxy_api_port: 8774
```

Comment:
All of these ports should just be defined in the service defaults.  There should not be a need to define them here.